independent-publisher-connectors/WebEx/apiDefinition.swagger.json

```diff
@@ -0,0 +1,291 @@
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "description": "Content-Type",
+            "x-ms-summary": "header"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "title": {
+                  "type": "string",
+                  "description": "title",
+                  "title": "",
+                  "x-ms-summary": "title"
+                  
+                },
+                "agenda": {
+                  "type": "string",
+                  "description": "agenda",
+                  "x-ms-summary": "agenda"
+                },
+                "password": {
+                  "type": "string",
+                  "description": "password",
+                  "x-ms-summary": "password"
+                },
+                "start": {
+                  "type": "string",
+                  "description": "start",
+                  "title": "",
+                  "x-ms-summary": "start_time"
+                },
+                "end": {
+                  "type": "string",
+                  "description": "end",
+                  "title": "",
+                  "x-ms-summary": "end_time"
+                },
+                "timezone": {
+                  "type": "string",
+                  "description": "timezone",
+                  "title": "",
+                  "default": "Europe/London",
+                  "x-ms-summary": "timezone"
+                },
+                "enabledAutoRecordMeeting": {
+                  "type": "string",
+                  "description": "enabledAutoRecordMeeting",
+                  "title": "",
+                  "default": "false",
+                  "x-ms-summary": "autorecord"
+                },
+                "allowAnyUserToBeCoHost": {
+                  "type": "string",
+                  "description": "allowAnyUserToBeCoHost",
+                  "title": "",
+                  "default": "false",
+                  "x-ms-summary": "cohost"
+                }
+              },
+              "default": {
+                "title": "Sample Title",
+                "agenda": "Sample Agenda",
+                "password": "A@ssword123",
+                "start": "{{_start_time}}",
+                "end": "{{_end_time}}",
+                "timezone": "{{TIMEZONE_STRING}}",
+                "enabledAutoRecordMeeting": false,
+                "allowAnyUserToBeCoHost": false
+              },
+              "required": [
+                "title",
+                "start",
+                "end"
+              ]
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/meetings/%7B%7B_singleMeetingId%7D%7D": {},
```

Comment:
@ocarhui can you please explain the empty actions? can you please remove them?
![image](https://user-images.githubusercontent.com/97551789/217964369-ace69e98-49cd-45ac-89e3-a93b4bab8e45.png)
![image](https://user-images.githubusercontent.com/97551789/217964479-cd74566e-1cdb-4add-8d41-92c1198a309c.png)